docs/rules/prefer-find-by.md

```diff
@@ -0,0 +1,78 @@
+# Use `find*` query methods to wait for elements instead of waitFor (prefer-find-by)
```

Comment:
Can you use here same description from README please?

```suggestion
# Suggest using `findBy*` methods instead of the `waitFor` + `getBy` queries (prefer-find-by)
```

Reply:
done